tests/vmi_multus_test.go

```diff
@@ -152,10 +154,12 @@ var _ = Describe("Multus", func() {
 	})
 
 	Describe("[rfe_id:694][crit:medium][vendor:cnv-qe@redhat.com][level:component]VirtualMachineInstance using different types of interfaces.", func() {
+		vmiOneCIDR := "10.1.1.1/24"
```

Comment:
In case of ptp, doesn't the cidr of the vm is "10.1.1.0/24" and 10.1.1.1 is the ip of the gateway (the host side of the veth)?